templates/docs/component-status.md

```diff
@@ -28,6 +28,11 @@ When we add, make significant updates, or deprecate a component we update their
       <td><div class="p-label--new">New</div></td>
       <td>2.28.0</td>
       <td>We added the optional footer to the modal pattern.</td>
+    <tr>
```

Comment:
Please add a line about deprecating inline images as well.
Also, add a note about deprecated pattern to our draft `upgrade-guide-v3.md`, so we don't forget to document it with major release.